src/main/resources/tags/links/uuid.tag

```diff
@@ -2,5 +2,16 @@ type: text
 aliases: xuid, floodgateuuid, bedrockuuid
 
 ---
+Automatic method:
+Use https://uuid.kejona.dev/ to convert any Bedrock Player username into a Floodgate UUID.
+Alternatively, run the "/UUID" command in #botspam.
+Manual method:
+You can manually find out a UUID using this method:
+Step 1: Visit https://www.cxkes.me/xbox/xuid and enter the bedrock username (without the period that is added by Geyser)
+Step 2: Search for the hexidecimal result and copy it down
+Step 3: You can try just that number instead of the username, but mine was formatted like this
 
-You can find a Bedrock player's UUID before they join using this (for whitelists for example): https://floodgate-uuid.heathmitchell1.repl.co/
+XUID Search Result: 000901F4BF5B7415
+Minecraft XUID: 00000000-0000-0000-0009-01f4bf5b7415
+
+So step 4 would be to add a dash after the first four digits and then paste "00000000-0000-0000-" before the search result to get the correct format.
```

Comment:
Can we remove this? There are 3 ways to get it automatically now so we are probably 3

Reply:
I've changed it to just show how a floodgate uuid is "formatted" from a XUID - or should i rather remove it altogether?